R/input-date.R

```diff
@@ -94,7 +94,8 @@
 dateInput <- function(inputId, label, value = NULL, min = NULL, max = NULL,
   format = "yyyy-mm-dd", startview = "month", weekstart = 0,
   language = "en", width = NULL, autoclose = TRUE,
-  datesdisabled = NULL, daysofweekdisabled = NULL) {
+  datesdisabled = NULL, daysofweekdisabled = NULL,
+  sassVariables = list()) {
```

Comment:
Yea, I'm not in love with the argument name. I was also thinking maybe `defaults` since it's intended for variable defaults and matches naming used in `sass::sass_layer()`.

I also really like the idea of adding `...`. Also, FWIW, I could maybe be swayed into not including the argument at all for this input, but it seems like it could be quite useful for `sliderInput()` and `selectInput()`,  so I think for consistency sake we should take similar approaches in all of them